src/test/create-user.test.ts

```diff
@@ -1,34 +1,89 @@
 import axios from 'axios';
 import { expect } from 'chai';
+import { AppDataSource } from '../data-source';
+import { passwordHashing } from '../resolvers';
+import { User } from '../User';
+import { EXPECTED_USER, MUTATION, TEST_URL } from './test-constants';
 
-describe('createUser mutation', () => {
-  it('should create a new user', async () => {
-    const expectedUser = {
-      data: {
+describe('User Tests', () => {
+  describe('createUser mutation', () => {
+    it('Should create a new user', async () => {
```

Comment:
Não acha que a árvore de testes fica verbosa fazendo um describe+it pra cada caso de teste. Geralmente temos um describe só pra uma query/mutation específica, e todos os seus casos de testes em um `it`. Dá uma olhada aqui pra mais boas práticas: https://www.notion.so/taqtile/Boas-pr-ticas-para-testes-66f8b1e317be4d88bde55604d45f8fe1